server/events/group.js

```diff
@@ -191,12 +217,17 @@ handleGroupOld.flush = function flushGroup() {
 
 function handleGroupNew(event, { hull }) {
   if (event && event.groupId) {
-    let scopedClient;
-    if (event.userId) {
-      scopedClient = hull.asUser({ external_id: event.userId })
-        .account({ external_id: event.groupId });
-    } else {
-      scopedClient = hull.asAccount({ external_id: event.groupId });
+    const scopedClient = event.userId
+      ? hull
+          .asUser({ external_id: event.userId })
+          .account({ external_id: event.groupId })
+      : hull.asAccount({ external_id: event.groupId });
```

Comment:
I'm not a big fan of the ?: syntax in any language.  But especially when there's more than 1 thing going on.  Yes, it uses up less space than if/else, but it's less clear to read